cmd/unmount.go

```diff
@@ -51,20 +51,13 @@ var unmountCmd = &cobra.Command{
 	SuggestFor:        []string{"unmount", "unmnt"},
 	Args:              cobra.ExactArgs(1),
 	FlagErrorHandling: cobra.ExitOnError,
-	RunE: func(cmd *cobra.Command, args []string) error {
+	RunE: func(_ *cobra.Command, args []string) error {
 		if strings.Contains(args[0], "*") {
 			mntPathPrefix := args[0]
 
-			lstMnt, err := common.ListMountPoints()
-			if err != nil {
-				return fmt.Errorf("failed to list mount points [%s]", err.Error())
-			}
-
+			lstMnt, _ := common.ListMountPoints()
```

Comment:
Why are we not returning errors on failure to complete these operations?

Reply:
List mount fails only in case we are not able to read through /etc/mtab, which shall never happen so its most likely a dead code which we will never hit and even our code coverage will report its never hitting. To avoid such cases we have ignored those error handlings.